Sources/Networking/Modifiers/Processors/EndpointRequestStorageProcessor/EndpointRequestStorageProcessor.swift

```diff
@@ -138,7 +141,7 @@ private extension EndpointRequestStorageProcessor {
         endpointRequest: EndpointRequest,
         urlRequest: URLRequest
     ) {
-        Task(priority: .background) { [weak self] in
+        Task.detached(priority: .utility) { [weak self] in
```

Comment:
Although I think logical to have it detached, I'm just wondering if it would work also without having it detached?

Reply:
Yes, it would work. This was @matejmolnar 's idea, which made sense to me, so I added that.